externals/pio2/cmake/FindPnetCDF.cmake

```diff
@@ -47,22 +47,22 @@ foreach (PNCDFcomp IN LISTS PnetCDF_FIND_VALID_COMPONENTS)
                               INCLUDE_DIRECTORIES ${MPI_${PNCDFcomp}_INCLUDE_PATH}
                               LIBRARIES ${MPI_${PNCDFcomp}_LIBRARIES})
             find_package_component(PnetCDF COMPONENT ${PNCDFcomp}
-                                   PATHS ${PnetCDF_${PNCDFcomp}_PATHS})
+                                   PATHS ${PnetCDF_${PNCDFcomp}_PATHS} ${NETCDF_DIR})
```

Comment:
I think this is the wrong place to do this.  I want to look at some alternatives.